frontend/src/components/activePage/TableTemplate/TableTemplate.tsx

```diff
@@ -65,6 +90,25 @@ const TableTemplate: FC<ITableTemplateProps> = ({ ...props }) => {
       ),
     },
   ];
+
+  useEffect(() => {
+    window.sessionStorage.setItem(
+      `prevExpandedIdActivePageTemplate-${templates[0].workspaceId}`,
+      expandedId.join(',')
+    );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [expandedId]);
+
+  useEffect(() => {
+    if (collapseAll) {
+      collapseTemplate();
+    }
+    if (expandAll) {
+      expandTemplate();
+    }
```

Comment:
```suggestion
    if (collapseAll) collapseTemplate();
    if (expandAll) expandTemplate();
```
stay compact, check other occurrences